test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertTo-Html.Tests.ps1

```diff
@@ -0,0 +1,153 @@
+Describe "ConvertTo-Html Tests" -Tags "CI" {
+
+    BeforeAll {
+        $customObject = [pscustomobject]@{"Name" = "John Doe"; "Age" = 42; "Friends" = ("Jack", "Jill")}
+        $newLine = "`r`n"
+    }
+
+    function normalizeLineEnds([string]$text)
+    {
+        $text -replace "`r`n?|`n", "`r`n"
+    }
+
+    It "Test ConvertTo-Html with no parameters" {
+        $returnObject = $customObject | ConvertTo-Html
+        $returnObject -is [System.Array] | Should Be $true
```

Comment:
I think the test is appropriate; however, it would be easier to diagnose if Type.FullName is compared instead of a soft-cast.